src/ocaml/merlin_specific/browse_raw.ml

```diff
@@ -769,10 +769,16 @@ let module_expr_paths { Typedtree. mod_desc } =
     [reloc (Path.Pident id) loc, Option.map mk_lident loc.txt]
   | _ -> []
 
+let bindop_path { bop_op_name; bop_op_path } =
+  let loc = bop_op_name
+  and path = bop_op_path in
```

Comment:
Nitpick: we generally do not use the `and` keyword for non-mutually-recursive definitions in Merlin's codebase. Multiple `let ... in` are preferred.

Reply:
Fixed.